app/assets/javascripts/deploys.js

```diff
@@ -225,16 +224,28 @@ $(function () {
   });
 
   // If there are messages being streamed, then show the output and hide buddy check
-  $('#messages').bind('contentchanged', function(){
+  $('#messages').bind('contentchanged', function() {
     var $output = $('#output');
     if ($output.find('.output').hasClass("hidden") ){
       $output.find('.output').removeClass('hidden');
       $output.find('.deploy-check').hide();
     }
   });
-
 });
 
 function toggleOutputToolbar() {
   $('.only-active, .only-finished').toggle();
 }
+
+function waitUntilEnabled(path) {
```

Comment:
is it strange to go from a stream to polling? Couldn't we just send down an enabled event?